docs/core/howto/tap.rst

```diff
@@ -310,6 +310,107 @@ For a full list of cred plugins supported, see :api:`twisted.plugins <twisted.pl
 
 
 
+Deploy your application using python packages
+---------------------------------------------
+
+
+
```

Comment:
The existing formatting in this file is an artifact of a conversion tool from a different format.  There's no real reason to try to mimic it.  One blank line separating sections is plenty.  Content should also be formatted following http://rhodesmill.org/brandon/2012/one-sentence-per-line/

Reply:
I see. I tried to stay in this style. I think changing this would be worth another ticket, as the diff will get quite hard to read with so many changes.